sys/nginx/dev/default.conf

```diff
@@ -0,0 +1,43 @@
+server {
```

Comment:
are we sure we need a different NGINX config for local dev? .. we should avoid this IMHO